### PR TITLE
add documentation fixes

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*
Documentation update for Cost Data Collection lab

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/300_Labs/300_Optimization_Data_Collection/1_Grant_permissions.md [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
 chapter: false
-weight: 2
+weight: 1
 pre: "<b>2. </b>"
 ---
 
-### Role for Management Account
+### Permissions
+
+We will need to install 2 things to ensure DataCollection account can collect information accross all accounts in the AWS Organization.
```
Comment (yprikhodko) [inline content/Cost/300_Labs/300_Optimization_Data_Collection/1_Grant_permissions.md:11]: Maybe rephrase to 'We will need to install 2 IAM roles' or 2 CloudFormation templates?

**Review by yprikhodko — APPROVED** [review]

File: content/Cost/300_Labs/300_Optimization_Data_Collection/1_Grant_permissions.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/300_Labs/300_Optimization_Data_Collection/1_Grant_permissions.md]: done
